src/Adapter/Product/Update/Filler/ProductBasicInformationPropertyFiller.php

```diff
@@ -0,0 +1,105 @@
+<?php
+/**
+ * Copyright since 2007 PrestaShop SA and Contributors
+ * PrestaShop is an International Registered Trademark & Property of PrestaShop SA
+ *
+ * NOTICE OF LICENSE
+ *
+ * This source file is subject to the Open Software License (OSL 3.0)
+ * that is bundled with this package in the file LICENSE.md.
+ * It is also available through the world-wide-web at this URL:
+ * https://opensource.org/licenses/OSL-3.0
+ * If you did not receive a copy of the license and are unable to
+ * obtain it through the world-wide-web, please send an email
+ * to license@prestashop.com so we can send you a copy immediately.
+ *
+ * DISCLAIMER
+ *
+ * Do not edit or add to this file if you wish to upgrade PrestaShop to newer
+ * versions in the future. If you wish to customize PrestaShop for your
+ * needs please refer to https://devdocs.prestashop.com/ for more information.
+ *
+ * @author    PrestaShop SA and Contributors <contact@prestashop.com>
+ * @copyright Since 2007 PrestaShop SA and Contributors
+ * @license   https://opensource.org/licenses/OSL-3.0 Open Software License (OSL 3.0)
+ */
+declare(strict_types=1);
+
+namespace PrestaShop\PrestaShop\Adapter\Product\Update\Filler;
+
+use PrestaShop\PrestaShop\Adapter\Tools;
+use PrestaShop\PrestaShop\Core\Domain\Product\Command\UpdateProductCommand;
+use Product;
+
+/**
+ * Fills product properties which can be considered as a basic product information
+ */
+class ProductBasicInformationPropertyFiller implements ProductUpdatablePropertyFillerInterface
```

Comment:
```suggestion
class BasicInformationFiller implements ProductFillerInterface
```

Reply:
I'm thinking of ways to reduce all the namings which are quite long:
- the method name in the interface is already `fillUpdatableProperties` so the UpdateableProperty part can be removed in class names and interface
- since we already are in product namespace having product prefix everywhere is a bit redundant (although I kept it in `ProductFillterInterface` to avoid confusion with future potential `FillerInterface` used for other entites